src/CMakeLists.txt

```diff
@@ -53,6 +53,9 @@ if (APPLE)
             mac/pxClipboardNative.mm mac/window.mm
             unix/rtMutexNative.cpp unix/rtThreadPoolNative.cpp
             rtFileCache.cpp rtHttpCache.cpp)
+     	if (SUPPORT_DUKTAPE)
+          set(PXCORE_FILES ${PXCORE_FILES} rtScriptDuk/debugger/duk_debugger_interface_unix.c)
```

Comment:
Let's create two subdirectories under rtScriptDuk/debugger/.  One should be called win the other should be called unix (following the same pattern as platform specific files in pxCore/src).  Let's have both files named the same (duk_debugger_interface.c) and put the unix version under rtScriptDuk/debugger/unix/ and the windows version under rtScriptDuk/debugger/win/.